GovUkDesignSystem/Attributes/DataBinding/GovUkDataBindingMandatoryIntErrorTextAttribute.cs

```diff
@@ -23,12 +23,15 @@ public GovUkDataBindingMandatoryIntErrorTextAttribute(string errorMessageIfMissi
         /// <br/>e.g. "[Full name] must be 2 characters or more"
         /// <br/>e.g. "[Median age] must be a number"
         /// </summary>
-        public string NameAtStartOfSentence { get; private set; }
+        public string NameAtStartOfSentence { get; protected set; }
 
         /// <summary>
         /// A complete sentence of the form: ‘Enter [whatever it is]’.
         /// <br/>For example, ‘Enter your first name’.
         /// </summary>
-        public string ErrorMessageIfMissing { get; private set; }
+        public virtual string ErrorMessageIfMissing { get; }
+        
+        public virtual string IsWholeNumberErrorMessage => "";
```

Comment:
I'm not keen on adding these properties to these classes that will never use them, but I don't think it's worth the effort to change it at the moment.
I'm not sure exactly what I'd do differently to simplify things, but perhaps the model binder could look for two different attribute types. I might even go further and create two different model binders (one localisable) with a shared base.